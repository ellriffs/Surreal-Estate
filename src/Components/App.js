
import '/Users/elliotrobertson/Desktop/projects/surreal-estate/src/Styles/App.css';

function App() {
  return (
    <div className="App">
      <title>Surreal Estate</title>
    </div>
  );
}

export default App;
