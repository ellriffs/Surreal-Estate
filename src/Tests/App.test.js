import { render, screen } from '@testing-library/react';
import App from '../Components/App';

describe(App, () => {
test('renders App in the DOM', () => {
  render(<App />)
  expect(<App />).toMatchSnapshot()
});

test('Contains Surreal Estate Text', () => {
  render(<App />)
  const linkElement = screen.getByText(/Surreal Estate/i)
  expect(linkElement).toBeInTheDocument()
})
})