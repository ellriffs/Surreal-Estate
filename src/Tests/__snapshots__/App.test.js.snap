// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App in the DOM 1`] = `<App />`;
